bower.json

```diff
@@ -36,7 +36,12 @@
     "bower": "*",
     "install": "^1.0.4",
     "kbase-ui-plugin-catalog": "^1.2.0",
-    "bootstrap": "3.3.7"
+    "bootstrap": "3.3.7",
+    "pdfmake": "^0.1.36",
```

Comment:
Is this needed? Are you making PDFs with this widget? It's in the paths, too, but not referenced anywhere else.